### PR TITLE
Fix travis

### PR DESCRIPTION
**Pull request by danielsfskim**
Here I included all the necessary code to travis.yml including,
- bundle exec rspec
- bundle exec rubocop
- encryped heroku's api key and allowed it to deploy to heroku

Although.. the test is not passing due to the rubocop errors. Not sure where to go from here

### CODE REVIEW

**Comment by mscoutermarsh** [comment]
It looks like it's not using the rubocop config, because it's throwing errors on things like "line longer than 80 chars"

File: config/.rubocop.yml [hunk 1]
```diff
@@ -1,8 +1,11 @@
+inherit_from: .rubocop_todo.yml
```
Comment (mscoutermarsh) [inline config/.rubocop.yml]: This file has to be in the root of the repo.

Once you move it, I think travis will be green. 😄 

`./.rubocop.yml`

Comment (mscoutermarsh) [inline config/.rubocop.yml]: It has to be in the very top level folder. Cannot be in `config`, needs to be 1 level higher.

**Comment by mscoutermarsh** [comment]
@danielsfskim I made a couple tweaks & merged this in using another branch 🙌

**Comment by danielsfskim** [comment]
@mscoutermarsh thanks for the help! The travis issue was really stumping me.. 
Just for future reference, what tweaks did you make to finally get it working? Much appreciated!

**Pull request closed without merging**